client/components/LearnerViewDirective.js

```diff
@@ -728,29 +728,39 @@ tie.directive('learnerView', [function() {
          * @param {string} questionId ID of question whose data will be loaded
          */
         var loadQuestion = function(questionId) {
-          clearFeedback();
           question = QuestionDataService.getQuestion(questionId);
           tasks = question.getTasks();
           currentTaskIndex = 0;
-          cachedCode = CodeStorageService.loadStoredCode(
+          cachedCode = LocalStorageService.loadStoredCode(
             questionId, language);
           $scope.title = question.getTitle();
           $scope.editorContents.code = (
             cachedCode || question.getStarterCode(language));
+          var reinforcement = ReinforcementObjectFactory.create();
+          $scope.reinforcementBullets = reinforcement.getBullets();
+          $scope.feedbackStorage = [];
+          var loadedFeedback = LocalStorageService.loadStoredFeedback(
+            questionId, language);
+          if (loadedFeedback) {
+            $scope.feedbackStorage.push({
+              feedbackParagraphs: loadedFeedback.feedbackParagraphs
+            });
+            $scope.reinforcementBullets = loadedFeedback.reinforcementBullets ?
```

Comment:
Common JS shorthand for default values:

         $scope.reinforcementBullets = loadedFeedback.reinforcementBullets || [];